onboarding/src/Components/hooks/PackagePage.js

```diff
@@ -143,7 +100,7 @@ export function savePackageDetails(handleSuccess, packageId, title, description)
 /**
  * Add page into Pages for Package (todo: set owner as a logged HR manager?);
  */
-export function addPage(handleSuccess, title, packageId, order, owner){
+export function addPage(handleSuccess, title, order, packageId, owner){
```

Comment:
This swappiness caused an error :hushed: - page was added to the wrong package :anguished: